tests/installation/ipxe_install.pm

```diff
@@ -132,6 +132,42 @@ END_BOOTSCRIPT
     set_ipxe_bootscript($bootscript);
 }
 
+sub set_bootscript_agama {
+    my $host = get_required_var('SUT_IP');
+    my $arch = get_required_var('ARCH');
+    my $mirror_http = get_required_var('MIRROR_HTTP');
+    my $openqa_hostname = get_var('OPENQA_HOSTNAME', 'openqa.suse.de');
+    my $agama_iso = get_required_var('ISO');
+
+    # trim all strings from variables to get rid of bogus whitespaces
+    $host =~ s/^\s+|\s+$//g;
+    $arch =~ s/^\s+|\s+$//g;
+    $mirror_http =~ s/^\s+|\s+$//g;
+    $openqa_hostname =~ s/^\s+|\s+$//g;
+    $agama_iso =~ s/^\s+|\s+$//g;
+
+    my $agama_live_iso = get_var("AGAMA_LIVE_ISO_URL", "http://$openqa_hostname/assets/iso/$agama_iso");
+    my $install = "root=live:$agama_live_iso live.password=$testapi::password";
+    my $kernel = "$mirror_http/boot/$arch/loader/linux";
+    my $initrd = "$mirror_http/boot/$arch/loader/initrd";
+
+    my $cmdline_extra = set_bootscript_agama_cmdline_extra();
+
+    my $bootscript = <<"END_BOOTSCRIPT";
+#!ipxe
+echo ++++++++++++++++++++++++++++++++++++++++++
+echo +++++++++ openQA agama ipxe boot +++++++++
+echo +    Host: $host
+echo ++++++++++++++++++++++++++++++++++++++++++
+
+kernel $kernel $install $cmdline_extra
+initrd $initrd
+boot
+END_BOOTSCRIPT
```

Comment:
This part looks quite like ```set_bootscript```.

Reply:
Yes, it is intended, as per https://progress.opensuse.org/issues/168850. sorry to say there are so many duplicated lines here, but let us keep current status since lot of changes will happen at agama beta phase.